src/Controller/CocktailApiController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\CocktailName;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class CocktailApiController extends AbstractController
```

Comment:
Controllers should be used only for receiving calls from routes. Call your service different name and move to different source. 
e.g. this could be named CocktailApiRequest